Gopkg.toml

```diff
@@ -35,7 +35,7 @@
 
 [[constraint]]
   name = "github.com/Peripli/service-broker-proxy"
-  version = "=0.2.0"
+  branch = "visibilities-brokers"
```

Comment:
Should be reverted when the **service-broker-proxy** PR is merged

Reply:
reference release